### PR TITLE
[SAP Cloud SDK] Update deprecated Notion of SAP Cloud Platform

### LINKED ISSUE #18039: Connect to OData Service on Cloud Foundry Using SAP Cloud SDK
Tutorials: https://developers.sap.com/tutorials/s4sdk-odata-service-cloud-foundry.html
--------------------------

Hello colleagues,

The tutorial still talks about "SAP Cloud Platform" which is a term deprecated in Q1 2021. Please update the descriptions.

Best regards,
Marco

### PR DESCRIPTION
**Pull request by Johannes-Schneider**
This PR replaces the outdated notion of the SAP Cloud Platform (SCP) with the new SAP Business Technology Platform branding in the SAP Cloud SDK related tutorials.

----

Fixes #18039

### CODE REVIEW

File: tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md [hunk 1]
```diff
@@ -207,11 +207,11 @@ Any `ODataException` thrown by the OData call is caught and logged, before retur
 
 [ACCORDION-BEGIN [Step 5: ](Deploying the Project)]
 
-Depending on your chosen archetype and SAP Cloud Platform setup you can deploy the project on either `SAP Cloud Platform Neo` or `SAP Cloud Platform CloudFoundry`.
+Depending on your chosen archetype and the BTP setup you can deploy the project on either `SAP Cloud Platform Neo` or `SAP Cloud Platform CloudFoundry`.
```
Comment (MatKuhr) [inline tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md:210]:
```suggestion
Depending on your chosen archetype and the BTP setup you can deploy the project on either `SAP BTP Neo` or `SAP BTP CloudFoundry`.
```

But I think we might as well remove the mention of Neo entirely

File: tutorials/s4sdk-cloud-foundry-sample-application/s4sdk-cloud-foundry-sample-application.md [hunk 2]
```diff
@@ -217,7 +217,7 @@ public class HelloWorldServlet extends HttpServlet
 
 The `HelloWorldServlet` extends `HttpServlet`, so this will be a HTTP endpoint that customers can visit. It is mapped to the `/hello` route using `@WebServlet("/hello")`. By overriding the function `doGet`, you define what happens when a client performs an HTTP GET request on the `/hello` route. In this case the endpoint simply writes a response containing **`Hello World!`**
 
-**Note**: The application code runs seamlessly in `SCP Neo` as well as `SCP Cloud Foundry`. The `SAP Cloud SDK` is compatible with both versions and provides mechanisms to seamlessly transfer code between both environments.
+**Note**: The application code runs seamlessly in `SAP Business Technology Platform Neo` as well as `SAP Business Technology Platform Cloud Foundry`. The `SAP Cloud SDK` is compatible with both versions and provides mechanisms to seamlessly transfer code between both environments.
```
Comment (MichaelCzcz) [inline tutorials/s4sdk-cloud-foundry-sample-application/s4sdk-cloud-foundry-sample-application.md:220]: Please start a note with the label "note".
You can find more information about notes in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21).

File: tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md [hunk 3]
```diff
@@ -239,11 +239,11 @@ Any `ODataException` thrown by the OData call is caught and logged, before retur
...
+In order for your application to run you need to provide it with information about your S/4HANA system. Your `BusinessPartnerService` needs an URL. Depending on the system you would like to use, you might need to define an authentication method as well. This information is provided in the form of a destination which we retrieve (in a multi-tenant environment) by transparently accessing the destination service of the BTP. This is done by retrieving the destination via the `DestinationAccessor` by name, in your case `MyErpSystem`. There are multiple ways to create such a destination. For now you will provide it to our application by creating an environment variable that holds the information. You will create such a variable now both for a local deployment and a deployment on the SAP BTP Cloud Foundry.
```
Comment (MichaelCzcz) [inline tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md:246]: "which we retrieve" - Pleae don't say "we" unless you are referring to SAP.

File: tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md [hunk 4]
```diff
@@ -369,7 +369,7 @@ As you can see, the properties `BusinessPartner` and `LastName` will be marked a
 
 [ACCORDION-BEGIN [Step 9: ](Systems.json and credentials)]
 
-If you run your application on SAP Cloud Platform, the SDK can simply read the ERP destinations from the destination service. However, since the tests should run locally, we need a way to supply our tests with an ERP destination.
+If you run your application on the BTP, the SDK can simply read the ERP destinations from the destination service. However, since the tests should run locally, we need a way to supply our tests with an ERP destination.
```
Comment (MichaelCzcz) [inline tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md:372]: Please don't say "we" unless you are referring to SAP.

File: tutorials/s4sdkjs-deploy-application-cloud-foundry/s4sdkjs-deploy-application-cloud-foundry.md [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
+>**Note:** If you don't have a [SAP Business Technology Platform](https://account.hana.ondemand.com/) account, you need to create one.
 
-In order to deploy our application, we first need to login to `Cloud Foundry` in `SAP Cloud Platform` using the **`cf` CLI**. First we need to set an `API` endpoint. The exact URL of this `API` endpoint depends on the region your `subaccount` is in. Open the [SAP Cloud Platform Cockpit](https://account.hana.ondemand.com/) and navigate to the `subaccount` you are planning to deploy your application to. Click on "Overview" on the left and you can see the URL of the `API` endpoint.
+In order to deploy our application, we first need to login to `Cloud Foundry` in `SAP Business Technology Platform` using the **`cf` CLI**. First we need to set an `API` endpoint. The exact URL of this `API` endpoint depends on the region your `subaccount` is in. Open the [SAP Business Technology Platform Cockpit](https://account.hana.ondemand.com/) and navigate to the `subaccount` you are planning to deploy your application to. Click on "Overview" on the left and you can see the URL of the `API` endpoint.
```
Comment (MichaelCzcz) [inline tutorials/s4sdkjs-deploy-application-cloud-foundry/s4sdkjs-deploy-application-cloud-foundry.md:22]: Please don't say "we" unless you are referring to SAP.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for your PR. There are some minor style issues that need to be fixed.
Also, please update the tags. The name space "products" is no longer valid, use "software-product" instead.

File: tutorials/s4sdk-cloud-foundry-sample-application/s4sdk-cloud-foundry-sample-application.md (hunk 2 above)
Comment (MichaelCzcz) [reply tutorials/s4sdk-cloud-foundry-sample-application/s4sdk-cloud-foundry-sample-application.md:220]: Left out a word.
Please don't start a note with the label "note".

File: tutorials/s4sdkjs-deploy-application-cloud-foundry/s4sdkjs-deploy-application-cloud-foundry.md (hunk 5 above)
Comment (MichaelCzcz) [reply tutorials/s4sdkjs-deploy-application-cloud-foundry/s4sdkjs-deploy-application-cloud-foundry.md:22]: @Johannes-Schneider Please address the requested changes.

**Comment by jmmargo** [comment]
Per Michael's request, it seems that the occurrence of "we" has not been fixed.  In content, I think you are referring to the user themselves.  In that case, I suggest a more active voice, such as "In order to deploy our application, first login to 'Cloud Foundry'...

File: tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md (hunk 4 above)
Comment (Johannes-Schneider) [reply tutorials/s4sdk-odata-service-neo/s4sdk-odata-service-neo.md:372]: We are planning to drop our tutorials for the Neo environment. Therefore, I wont update this any more.

**Pull request merged**